tests/config_test.go

```diff
@@ -232,6 +233,45 @@ func Test_Configs(t *testing.T) {
 		ctx.validateDateFieldsForCreate(now, entityJson)
 
 		ctx.AdminSession.requireDeleteEntity(config)
-		ctx.requireNotFoundError(ctx.AdminSession.query("configs/" + uuid.New().String()))
+		ctx.requireNotFoundError(ctx.AdminSession.query("configs/" + config.id))
+	})
+
+	ctx.enabledJsonLogging = true
```

Comment:
Expected?

Reply:
Missed that. I've removed it for the next pass.